packages/manager/src/features/Kubernetes/kubeUtils.test.ts

```diff
@@ -67,6 +69,31 @@ describe('helper functions', () => {
       });
     });
   });
+  describe('APL availability', () => {
+    afterEach(() => {
+      vi.resetAllMocks();
+    });
+    it('should return true if apl flag is true and beta is active', () => {
+      vi.mock('@tanstack/react-query', () => ({
+        useQuery: vi.fn().mockReturnValue({
+          data: {
+            ended: '2099-01-01T00:00:00Z',
+            enrolled: '2023-01-15T00:00:00Z',
+            id: 'apl',
+            label: 'APL Beta',
+            started: '2023-01-01T00:00:00Z',
+          },
+          error: null,
+          isLoading: false,
+        }),
+      }));
+      vi.spyOn(useFlags, 'useFlags').mockReturnValue({ apl: true });
+
+      const aplAvailability = useGetAPLAvailability();
+
+      expect(aplAvailability).toBe(true);
+    });
```

Comment:
Because `useGetAPLAvailability` uses React Hooks, it might be more appropriate to unit test with the `renderHook` util

```suggestion
    it('returns true when the header is included', async () => {
      const accountBeta = accountBetaFactory.build({
        enrolled: '2023-01-15T00:00:00Z',
        id: 'apl',
      });

      server.use(
        http.get('*/account/betas/apl', () => {
          return HttpResponse.json(accountBeta);
        })
      );

      const { result } = renderHook(() => useGetAPLAvailability(), {
        wrapper: (ui) => wrapWithTheme(ui, { flags: { apl: true } }),
      });

      await waitFor(() => {
        expect(result.current).toBe(true);
      });
    });
```